### PR TITLE
enable reading and writing of bigints (int64) from tables

### PR DESCRIPTION
**Pull request by tantaman**

### CODE REVIEW

File: src/examples/IDBBatchAtomicVFS.js [hunk 1]
```diff
@@ -143,6 +143,7 @@ export class IDBBatchAtomicVFS extends VFS.Base {
   xRead(fileId, pData, iOffset) {
     return this.handleAsync(async () => {
       const file = this.#mapIdToFile.get(fileId);
+      iOffset = Number(iOffset)
```
Comment (tantaman) [inline src/examples/IDBBatchAtomicVFS.js:147]: this should never go out of bounds of `Number` but we should range check just in case.

**Comment by tantaman** [comment]
todo -- add some tests that read and write actual 64bit values that are out of range of Number.MAX and MIN SAFE INTEGER.
